great_tables/_data_color/base.py

```diff
@@ -563,17 +558,8 @@ def _expand_short_hex(hex_color: str) -> str:
     # Get the hex color without the leading '#'
     hex_color = hex_color[1:]
 
-    # Get the first character of the hex color
-    first_char = hex_color[0]
-
-    # Get the second character of the hex color
-    second_char = hex_color[1]
-
-    # Get the third character of the hex color
-    third_char = hex_color[2]
-
     # Return the expanded 6-digit hexadecimal color value
-    expanded = "#" + first_char + first_char + second_char + second_char + third_char + third_char
+    expanded = "".join(chain("#", *zip(*repeat(hex_color, 2))))
```

Comment:
What about something like this?:

```pyhton
"#" + "".join([x*2 for x in hex_color])
```

It took me a bit of thinking to think through how zip and repeat combined (since I don't use repeat too often 😅 )

Reply:
No worries. The updated code came to mind while refactoring. I believe your solution works and is arguably more readable. Personally, I might write `"#" + "".join(x * 2 for x in hex_color)`, but feel free to update this PR based on your judgment.